commands/init/mod.ts

```diff
@@ -95,24 +96,37 @@ export const init = command("init", {
     );
 
     const existingLinks = await config.get("projects");
-    let app: paths["/projects/{id}"]["get"]["responses"]["200"]["content"][
+    let app: paths["/apps/{id}"]["get"]["responses"]["200"]["content"][
       "application/json"
     ];
     const link = existingLinks[dest];
 
     if (!link) {
       logger.info(`Project doesn't exist. Creating a new one.`);
-      const res = await loading(projects.create({ name: appName }), {
-        text: "Creating app",
-        enabled: !flags.json,
-      });
+      const res = await loading(
+        apps.create({
+          config: {
```

Comment:
When the template is downloaded it comes packaged with a config. There's an app config helper that will find it for you:
https://github.com/Paperspace/cli/blob/c4bbfdfa6e804f92edc2e4acdae46d2b1575b818/lib/app-config.ts#L6

You should throw if one is not found.

Reply:
Actually that helper will throw for you